addon/templates/components/nypr-floating-banner.hbs

```diff
@@ -0,0 +1,12 @@
+{{#unless hideBanner}}
+  <div class="nypr-floating-banner__wrapper{{if arrow ' nypr-floating-banner__with-arrow'}}">
+    {{#nypr-card class='nypr-floating-banner'}}
+      
+      {{yield}}
+    
+      <button class="nypr-floating-banner__close" {{action (action (mut hideBanner) true)}}>
```

Comment:
huh, does this just create a hideBanner variable out of nothing when it's clicked? Learn something new every day.